schema/properties_directory/structural/basis/atomic_coordinate_value.json

```diff
@@ -0,0 +1,19 @@
+{
+    "schemaId": "properties-directory-structural-basis-atomic-coordinate-value",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "atomic coordinates",
+    "description": "coordinates of atoms by ids, vector, unitless",
+    "properties": {
+        "id": {
+            "type": "number"
+        },
+        "value": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            },
+            "minItems": 3,
+            "maxItems": 3
+        }
+    }
+}
```

Comment:
Maybe `"$ref": "../../../core/reusable/atomic_vectors.json"`?

Reply:
atomic_vectors.json expect an array of arrays of numbers like `[[1,2,3], [1,2,3]]` while we need only an array of numbers here:`[1,2,3]`. So instead of vector, I have changed it to `core/primitive/array_of_3_numbers.json`